assets/dev/js/editor/document/elements/commands/import.js

```diff
@@ -19,15 +19,15 @@ export class Import extends CommandHistory {
 
 	apply( args ) {
 		const { data, options = args.options || {} } = args,
-			previewContainer = elementor.getPreviewContainer(),
+			target = options.target || elementor.getPreviewContainer(),
```

Comment:
Why not to call the option `container` instead of `target`?
That's kinda our convention.

Reply:
Handled.